samples/unit-tests/drilldown/resetzoom/demo.js

```diff
@@ -155,6 +155,11 @@ QUnit.test('Drilldown and reset zoom', function (assert) {
         'object',
         'We have a drillUp button'
     );
+    assert.strictEqual(
+        typeof chart.resetZoomButton,
+        'object',
+        'Reset zoom button should also be visible.'
+    );
 
```

Comment:
In addition to checking for the button's existence, it would probably be a good idea to check that it does not overlap with the drillUp button.